features/web-audio.yml.dist

```diff
@@ -0,0 +1,732 @@
+  - api.BaseAudioContext.createConvolver
+  - api.BaseAudioContext.createDelay
+  - api.BaseAudioContext.createDynamicsCompressor
+  - api.BaseAudioContext.createGain
+  - api.BaseAudioContext.createIIRFilter
+  - api.BaseAudioContext.createOscillator
+  - api.BaseAudioContext.createPanner
+  - api.BaseAudioContext.createPeriodicWave
+  - api.BaseAudioContext.createStereoPanner
+  - api.BaseAudioContext.createWaveShaper
+  - api.BaseAudioContext.currentTime
+  - api.BaseAudioContext.decodeAudioData
+  - api.BaseAudioContext.destination
+  - api.BaseAudioContext.listener
+  - api.BaseAudioContext.sampleRate
+  - api.BaseAudioContext.state
+  - api.BaseAudioContext.statechange_event
+
+  # baseline: high
+  # baseline_low_date: 2021-04-26
+  # baseline_high_date: 2023-10-26
+  # support:
+  #   chrome: "57"
+  #   chrome_android: "57"
+  #   edge: "79"
+  #   firefox: "53"
+  #   firefox_android: "53"
+  #   safari: "14.1"
+  #   safari_ios: "14.5"
+  - api.MediaStreamAudioDestinationNode.MediaStreamAudioDestinationNode
+
+  # baseline: high
+  # baseline_low_date: 2021-04-26
+  # baseline_high_date: 2023-10-26
+  # support:
+  #   chrome: "57"
+  #   chrome_android: "57"
+  #   edge: "79"
+  #   firefox: "70"
+  #   firefox_android: "79"
+  #   safari: "14.1"
+  #   safari_ios: "14.5"
+  - api.AudioContext.getOutputTimestamp
+
+  # baseline: high
+  # baseline_low_date: 2021-04-26
+  # baseline_high_date: 2023-10-26
+  # support:
+  #   chrome: "58"
+  #   chrome_android: "58"
+  #   edge: "79"
+  #   firefox: "70"
+  #   firefox_android: "79"
+  #   safari: "14.1"
+  #   safari_ios: "14.5"
+  - api.AudioContext.baseLatency
+
+  # baseline: high
+  # baseline_low_date: 2021-04-26
+  # baseline_high_date: 2023-10-26
+  # support:
+  #   chrome: "64"
+  #   chrome_android: "64"
+  #   edge: "79"
+  #   firefox: "76"
+  #   firefox_android: "79"
+  #   safari: "14.1"
+  #   safari_ios: "14.5"
+  - api.AudioWorkletProcessor
+  - api.AudioWorkletProcessor.AudioWorkletProcessor
+  - api.AudioWorkletProcessor.port
+
+  # baseline: high
+  # baseline_low_date: 2021-04-26
+  # baseline_high_date: 2023-10-26
+  # support:
+  #   chrome: "66"
+  #   chrome_android: "66"
+  #   edge: "79"
+  #   firefox: "76"
+  #   firefox_android: "79"
+  #   safari: "14.1"
+  #   safari_ios: "14.5"
+  - api.AudioParamMap
+  - api.AudioWorklet
+  - api.AudioWorkletGlobalScope
+  - api.AudioWorkletGlobalScope.currentFrame
+  - api.AudioWorkletGlobalScope.currentTime
+  - api.AudioWorkletGlobalScope.registerProcessor
+  - api.AudioWorkletGlobalScope.sampleRate
+  - api.AudioWorkletNode
+  - api.AudioWorkletNode.AudioWorkletNode
+  - api.AudioWorkletNode.parameters
+  - api.AudioWorkletNode.port
+  - api.AudioWorkletNode.processorerror_event
+  - api.BaseAudioContext.audioWorklet
+
+  # baseline: false
+  # support:
+  #   chrome: "41"
+  #   chrome_android: "41"
+  #   edge: "14"
+  #   safari: "14.1"
+  #   safari_ios: "14.5"
+  - api.OfflineAudioContext.suspend
```

Comment:
This is because of https://bugzilla.mozilla.org/show_bug.cgi?id=1265406. The reason that `resume()` isn't listed here is that it's on `BaseAudioContext` in Gecko source, but should be on `AudioContext` and `OfflineAudioContext` separately per spec. I'm not sure if calling `resume()` can ever do anything.

Reply:
Turns out that `resume()` always rejects:

https://github.com/mozilla/gecko-dev/blob/bb682ab37e12bf256f4180693568bf1325aab2be/dom/media/webaudio/AudioContext.cpp#L1062-L1066

Reply:
I've sent https://github.com/mdn/browser-compat-data/pull/23641, which should end up moving `resume()` into this block.